tests/handlers.rs

```diff
@@ -392,7 +392,7 @@ speculate! {
 
                 assert_eq!(
                     signatures.len(),
-                    4,
```

Comment:
can you help me understand why this grew to `64`?

Reply:
the number of possible arguments increased for `from` so before we had 2 which results in 4 possible combinations now there are 6 arguments which is 6 possible combinations